Libraries/WNScripting/CMakeLists.txt

```diff
@@ -7,6 +7,17 @@ set(WN_LLVM_TRIPLE)
 set(WN_LLVM_TARGET)
 set(WN_LLVM_ADDITIONAL_BUILD_FLAGS)
 set(WN_LLVM_OVERRIDDEN_TRIPLE "WN_LLVM_NON_OVERRIDDEN_TRIPLE")
+set(WN_CXX_FLAGS_DEBUG=${CMAKE_CXX_FLAGS_DEBUG})
```

Comment:
Why did we need to do this? Also does it need the `=`?

Reply:
Is this just so we can disable debug stuff for only LLVM? Just want to make sure I get what you're doing

Reply:
oops dont need the =
I will fix the = that is wrong
But in general, this so that overlays can modify these flags. (To remove flags for example)